src/Modules/Workspaces/Commands.Workspaces.Test/GetPowerBIWorkspaceTests.cs

```diff
@@ -48,6 +48,80 @@ public void EndToEndGetWorkspacesOrganizationScope()
             }
         }
 
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void EndToEndGetWorkspacesAllAndOrganizationScope()
+        {
+            /*
+             * Test requires at least one workspace (group or preview workspace) and login as an administrator
+             */
+            using (var ps = System.Management.Automation.PowerShell.Create())
+            {
+                // Arrange
+                ProfileTestUtilities.ConnectToPowerBI(ps);
+                var parameters = new Dictionary<string, object>()
+                    {
+                        { nameof(GetPowerBIWorkspace.Scope), PowerBIUserScope.Organization },
+                        { nameof(GetPowerBIWorkspace.All), new SwitchParameter(true) }
```

Comment:
I believe you can just pass try here as PS will autobox the type, but this is minor